pywinauto/win32functions.py

```diff
@@ -31,250 +31,672 @@
 
 """Defines Windows(tm) functions"""
 
-import ctypes
+from ctypes import windll
 from ctypes import wintypes
 from . import win32defines, win32structures
 from .actionlogger import ActionLogger
-from ctypes import c_uint, c_short, c_long
+from ctypes import c_short
+from ctypes import c_long
+from ctypes import WINFUNCTYPE
+from ctypes import c_void_p
+from ctypes import c_int
+from ctypes import byref
+from ctypes import POINTER
+from ctypes import c_ubyte
+from ctypes import c_size_t
+from ctypes import cdll
 
 import sys
 if sys.platform == "cygwin":
-    windll = ctypes.cdll
+    cygdll = cdll

```

Comment:
`cygdll` is not used. Do we really need this rename?

Reply:
This reference to cygwin was added long time ago by Mark, but I also don't see how it was used. All the API declarations below always refered to ```ctypes.windll``` explicitly. I'll drop this code for now. Cygwin support, if it's still required, will need a separate issue.